admin-dev/themes/new-theme/js/pages/order/create/create-order-page.js

```diff
@@ -173,6 +176,15 @@ export default class CreateOrderPage {
       'width': '90%',
       'height': '90%',
     });
+
+  }
+
+  initCartRuleButtonsIframe() {
+    $('#js-add-cart-rule-btn').fancybox({
```

Comment:
Noticed a bit too late, but this needs to use mapped selector instead of hardcoded one :P

Reply:
Thinking to create a new PR for this change